ports/rp2/boards/WAVESHARE_RP2040_PLUS_16MB/mpconfigboard.h

```diff
@@ -0,0 +1,10 @@
+// url   :   https://www.waveshare.com/product/rp2040-plus.htm
+// wiki  :   https://www.waveshare.com/wiki/RP2040-Plus
+//  Board and hardware specific configuration
+#define MICROPY_HW_BOARD_NAME "Waveshar RP2040-Plus"
+#define MICROPY_HW_FLASH_STORAGE_BYTES (3 * 1024 * 1024)
+
+#define MICROPY_HW_USB_VID (0x2E8A)
+#define MICROPY_HW_USB_PID (0x1020)
+
+// user led gpio25
```

Comment:
should this go in a pins.csv?  there's no point in just having a dangling comment